dolphinscheduler-server/src/test/java/org/apache/dolphinscheduler/server/master/DependentTaskTest.java

```diff
@@ -37,188 +45,426 @@
 import org.slf4j.LoggerFactory;
 import org.springframework.context.ApplicationContext;
 
-import java.util.ArrayList;
-import java.util.Date;
-import java.util.List;
-
 @RunWith(MockitoJUnitRunner.Silent.class)
```

Comment:
```suggestion
@RunWith(MockitoJUnitRunner.class)
```

I test in my local env, you can use `MockitoJUnitRunner.class` to replace `MockitoJUnitRunner.Silent.class`

Reply:
> I test in my local env, you can use `MockitoJUnitRunner.class` to replace `MockitoJUnitRunner.Silent.class`

It’s okay to run this test alone, but the overall run may be because the mock is defined elsewhere, which leads to an exception. Therefore, I suggest that he restore temporarily.

Reply:
I found it's necessary to use the "Slient" one here. For other tests raised problems and leaded `Stopper.isRunning() == false`, then the mockito treated it as unused and triggered `UnnecessaryStubbingException`. All things unusual leads to the unit test build failed. Wired but not involved in this PR.

Reply:
It may also uncovered that tests in a branch of `Stopper.isRunning() == true` are ignored ?

Reply:
Yes, so it is recommended to ignore it first. In fact, we are currently not independent enough for single test, and there are too many cross tests. If you are interested, you can try to refactor it.

Reply:
> > I test in my local env, you can use `MockitoJUnitRunner.class` to replace `MockitoJUnitRunner.Silent.class`
> 
> It’s okay to run this test alone, but the overall run may be because the mock is defined elsewhere, which leads to an exception. Therefore, I suggest that he restore temporarily.

Got it.